contents/docs/error-tracking/index.mdx

```diff
@@ -75,8 +75,13 @@ Should an exception be autocaptured you will need to modify the properties befor
 ```javascript
 posthog.init("<API_KEY>", {
   before_send: (event) => {
-    if (event.event === "$exception" && event.properties["$exception_list"][0]["$exception_type"] == "SyntaxError") {
-      event.properties["$exception_fingerprint"] = "MyCustomGroup"
+    if (event.event === "$exception") {
```

Comment:
could also do a pre-baked one in the customizations file once you have a good feel for what people want to do (or if you want to control what they do)

Reply:
a dedupe baked-in is a good default btw, its common that people:

```
try {
  1 / 0
} catch (e) {
  capture(e)
  rethrow
}
```
// so the event will be captured twice. 1. by capture(e) and 2. by the signal handler when `rethrow`

Reply:
> could also do a pre-baked one in the customizations file once you have a good feel for what people want to do (or if you want to control what they do)

yes, its always nice having those, just be aware of minification, `exception["$exception_type"] == "SyntaxError")` won't work for minified apps because `SyntaxError` is now `abc`, usually it's better doing class type comparisons.

like `exception["$exception_instance"] instanceof SyntaxError`